packages/tectonic-explorer/js/components/data-collection-dialog.tsx

```diff
@@ -59,7 +60,7 @@ export const DataCollectionDialog = ({ currentDataSample, onClose, onSubmit, onN
         </table>
         {
           notesEnabled &&
-          <textarea key="notes" placeholder="Add notes…" value={currentDataSample.notes} onChange={handleNotesChange} />
+          <textarea placeholder="Add notes…" value={currentDataSample.notes} onChange={handleNotesChange} />
```

Comment:
With the addition (in the other PR) of the `id` field, would a key of "notes_" + currentDataSample.id be good to add here or is this dialog modal so that `currentDataSample` doesn't change while it is displayed?